setup.py

```diff
@@ -215,41 +133,27 @@
     'scripts/standardizeModelSpeciesNames.py',
     'scripts/thermoEstimator.py',
     'scripts/isotopes.py',
-    'testing/databaseTest.py',
 ]
 
-modules = []
-for root, dirs, files in os.walk('rmgpy'):
-    if 'test_data' in root:
-        continue
-    for f in files:
-        if f.endswith('.py') or f.endswith('.pyx'):
-            if 'Test' not in f and '__init__' not in f:
-                module = 'rmgpy' + root.partition('rmgpy')[-1].replace('/', '.') + '.' + f.partition('.py')[0]
-                modules.append(module)
-for root, dirs, files in os.walk('arkane'):
-    if 'data' in root:
-        continue
-    for f in files:
-        if f.endswith('.py') or f.endswith('.pyx'):
-            if 'Test' not in f and '__init__' not in f:
-                module = 'arkane' + root.partition('arkane')[-1].replace('/', '.') + '.' + f.partition('.py')[0]
-                modules.append(module)
-
 # Read the version number
 exec(open('rmgpy/version.py').read())
 
 # Initiate the build and/or installation
 setup(
-    name='RMG-Py',
+    name='reactionmechanismgenerator',
```

Comment:
Is this renaming a significant (possibly breaking) change?
what are the consequences and what did it enable?